packages/contracts/test/token/erc20/governance-wrapped-erc20.ts

```diff
@@ -61,35 +64,42 @@ describe('GovernanceWrappedERC20', function () {
 
   beforeEach(async function () {
     defaultExistingERC20InitData = [existingErc20Name, existingErc20Symbol, 0];
-    erc20 = await TestERC20.deploy(...defaultExistingERC20InitData);
+    erc20 = await hre.wrapper.deploy('TestERC20', {
+      args: defaultExistingERC20InitData,
+    });
+
+    dao = await deployNewDAO(signers[0]);
```

Comment:
This DAO is not used in the tests. Please delete.

Reply:
https://github.com/aragon/osx/pull/596/commits/6710f4efa85095b3a2451c25180643479940cb66